File: Selenium/Activities/Activity_5.java
package test_basic_practice;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.interactions.Actions;

public class Activity_5 {

	public static void main(String[] args) {
		WebDriver drive =new ChromeDriver();
		drive.get("https://v1.training-support.net/selenium/input-events");
		System.out.println("Home page Title is: "+drive.getTitle());
		Actions action =new Actions(drive);
		//left click
		
		action.click().pause(1000).build().perform();
		String cubeValue_left=drive.findElement(By.className("active")).getText();
		System.out.println("left click cube value is: "+cubeValue_left);
		//double click
		
		action.doubleClick().pause(1000).build().perform();
		String cubeValue_double=drive.findElement(By.className("active")).getText();
		System.out.println("double click cube value is: "+cubeValue_double);
		
		//right click
		action.contextClick().pause(1000).build().perform();
		String cubeValue_right=drive.findElement(By.className("active")).getText();
		System.out.println("right click cube value is: "+cubeValue_right);
		
		drive.quit();
	}

}

File: Selenium/Activities/Activity_5.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support.ui import WebDriverWait
from selenium.webdriver.support import expected_conditions

with webdriver.Firefox() as driver:
    driver.get("https://v1.training-support.net/selenium/input-events")
    print("Title is: ",driver.title)
    wait = WebDriverWait(driver, 10)
    actions = webdriver.ActionChains(driver)
    actions.click().perform()
    cubeValue_left=driver.find_element(By.CLASS_NAME,"active").text
    print("left click cube value is: ",cubeValue_left)
    actions.double_click().perform()
    cubeValue_double=driver.find_element(By.CLASS_NAME,"active").text
    print("double click cube value is: ",cubeValue_double)
    actions.context_click().perform()
    cubeValue_right=driver.find_element(By.CLASS_NAME,"active").text
    print("right click cube value is: ",cubeValue_right)
		